tracer/test/test-applications/integrations/Samples.Microsoft.Data.SqlClient/Samples - Backup.Microsoft.Data.SqlClient.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
not sure about that backup file?

Reply:
Yeah, should have deleted that. Not sure why VS keeps creating them.